integrations/gitlab/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gitlab"
-version = "0.2.72"
+version = "0.2.72.post1"
```

Comment:
```suggestion
version = "0.2.72-post"
```

Reply:
or `-post1` if just `post` isn't supported

Reply:
but lets make sure that in the bumps we aren't bumping the `post1 -> post2` but rather its prefix version